tests/contrib/requests/test_requests.py

```diff
@@ -109,23 +124,13 @@ def test_non_existant_url():
         assert "Traceback (most recent call last)" in s.get_tag(errors.STACK)
         assert "requests.exception" in s.get_tag(errors.TYPE)
 
-
-    @staticmethod
-    def test_500():
-        tracer, session = get_traced_session()
-        out = session.get(URL_500)
+    def test_500(self):
+        out = self.session.get(URL_500)
         eq_(out.status_code, 500)
 
-        spans = tracer.writer.pop()
+        spans = self.tracer.writer.pop()
         eq_(len(spans), 1)
         s = spans[0]
         eq_(s.get_tag(http.METHOD), 'GET')
         eq_(s.get_tag(http.STATUS_CODE), '500')
         eq_(s.error, 1)
-
-
-def get_traced_session():
```

Comment:
Removed since a not monkey-patched class was used. With this new test suite, we ensure that `patch()` really updates the `Session` class.